src/pages/solve/SAT.tsx

```diff
@@ -1,80 +1,65 @@
 import Head from "next/head";
 import React, { useState } from "react";
 import type { NextPage } from "next";
-import { Container } from "../../components/Container";
-import { Main } from "../../components/Main";
-import { SolverTitle } from "../../components/solvers/SolverTitle";
 import { TextArea } from "../../components/solvers/SAT/TextArea";
-import { Help } from "../../components/solvers/SAT/Help";
-import { InputButtonPanel } from "../../components/solvers/buttons/InputButtonPanel";
 import { ProgressHandler } from "../../components/solvers/ProgressHandler";
-import { Text, Divider } from "@chakra-ui/react";
+import { Text, Divider, Heading, Spacer } from "@chakra-ui/react";
 import { DimacsParser } from "../../converter/dimacs/DimacsParser";
 import { LogicalExpressionParser } from "../../converter/dimacs/LogicalExpressionParser";
+import { Layout } from "../../components/layout/Layout";
+import { EditorControls } from "../../components/solvers/EditorControls";
+import { baseUrl } from "../../api/ToolboxAPI";
 
 const SAT: NextPage = () => {
   const logicalExpressionParser = new LogicalExpressionParser();
-  const dimacsParser = new DimacsParser();
 
   const [logicalExpressionString, setLogicalExpressionString] = useState("");
   const [errorString, setErrorString] = useState("");
 
   return (
-    <Container minHeight="100vh">
+    <Layout>
       <Head>
         <title>ProvideQ</title>
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
         {/* TODO: replace favicon */}
       </Head>
-      <SolverTitle
-        title="SAT Solver"
-        text="For a given Boolean formula, this algorithm checks if there exists an interpretation that satisfies it."
+
+      <Heading as="h1">SAT Solver</Heading>
+      <Text color="text" align="justify">
+        For a given Boolean formula, this algorithm checks if there is an
+        interpretation that satisfies it. You can enter any boolean formula with
+        any number of variables and combine them with boolean operators.
+      </Text>
+
+      <Spacer />
+
+      <EditorControls
+        idleText={'Try "a and not (not a or not b)" 👇'}
```

Comment:
```suggestion
        idleText={'Try "a and (not a or not b)" 👇'}
```

Reply:
Currently the website has a problem this with formula

Reply:
btw, this is tracked in https://github.com/ProvideQ/toolbox-web/issues/29